graylog2-server/src/main/java/org/graylog2/audit/AuditEventTypes.java

```diff
@@ -172,6 +172,10 @@ public class AuditEventTypes implements PluginAuditEventTypes {
     public static final String TELEMETRY_USER_SETTINGS_UPDATE = PREFIX + "telemetry_user_settings:update";
     public static final String CONTENT_STREAM_USER_SETTINGS_UPDATE = PREFIX + "content_stream_user_settings:update";
 
+    public static final String ENTITY_GROUP_CREATE = PREFIX + "category:create";
+    public static final String ENTITY_GROUP_UPDATE = PREFIX + "category:update";
+    public static final String ENTITY_GROUP_DELETE = PREFIX + "category:delete";
```

Comment:
I think this might have been missed in the recent changes, but I believe we were going to use the name `entity_group` everywhere.
```suggestion
    public static final String ENTITY_GROUP_CREATE = PREFIX + "entity_group:create";
    public static final String ENTITY_GROUP_UPDATE = PREFIX + "entity_group:update";
    public static final String ENTITY_GROUP_DELETE = PREFIX + "entity_group:delete";
```